src/api/app/models/bs_request.rb

```diff
@@ -888,17 +888,6 @@ def reviews_for_user_and_others(user)
   def webui_infos(opts = {})
     opts.reverse_merge!(diffs: true)
     result = {}
-    result['id'] = id
-    result['number'] = number
-
-    result['description'] = description
-    result['priority'] = priority
-    result['state'] = state
-    result['creator'] = User.find_by_login(creator)
-    result['created_at'] = created_at
-    result['accept_at'] = accept_at if accept_at
-    result['superseded_by'] = superseded_by if superseded_by
-    result['superseding'] = superseding unless superseding.empty?
     result['is_target_maintainer'] = is_target_maintainer?(User.current)
```

Comment:
did you look if the webui_info is used in another places? i.e ```BsRequest.truncated_diffs?(@req)```

Reply:
I am not sure, what you mean, can you please elaborate? :confused:

Reply:
If you mean that `@req` is used in `truncated_diffs?`. This is another case where  a hash with many info that is not needed is passed as parameter. This shouldn't do like this because of many reasons, for example performance and maintainability. You can easily solve this by passing the two variables. Anyway, this was a _nitpick_, maybe it was not clear.

Reply:
I would prefer to tackle the remaining variables in a separate step. Moving this to the controller just to get rid of this method doesn't seem worth it.
Especially when we are going to remove the code anyway (either from here or the controller you want to move it to).

Reply:
@vpereira 

> I would prefer to tackle the remaining variables in a separate step. Moving this to the controller just to get rid of this method doesn't seem worth it.
Especially when we are going to remove the code anyway (either from here or the controller you want to move it to).

You convinced me, let leave this by now :+1: